### ISSUE TITLE
Test performance reported in paper not available when running experiments

### ISSUE THREAD

**Issue by TrungQuocNguyen** [LUOyk1999/HDSE#2]
Dear the author of HDSE, 

Thank you for this novel work. I really enjoy it when reading your method. However, when I ran the experiments with GraphGPS+HDSE, as instructed, the folder 'agg' in  'HDSE/graph-level/GraphGPS+HDSE/results/mnist-GPS-ESLapPE10/agg' only contains results for training and validation set. The result or test set is nowhere to be found. The same goes with other datasets. Have I not found it yet, or there is no test performance reported in the experiments ? 

Thank you.

**Comment by LUOyk1999** [comment]
Hi! Thank you for your question!

The test set results should be in 'HDSE/graph-level/GraphGPS+HDSE/results/mnist-GPS-ESLapPE10/0/logging.log', assuming the seed used is 0. 
The 'agg' folder is not where the test set results are stored.
